challenges/2022-06-14-two-sum-numbers/solutions/javascript/Moses-Alero/TwosumNumbers.js

```diff
@@ -0,0 +1,11 @@
+const TwoSumNumbers = function (number, numbersArray){
+
+    for(let i = 0; i< numbersArray.length; i++){
+        const summation = number - numbersArray[i];
+        numbersArray.shift(numbersArray[i]);
```

Comment:
What are you hoping to do here? The Array.shift() method does not take any arguments. It just removes the first element of an array.

Reply:
It's to help prevent a situation whereby
 `summation = number - numbersarray[i]` equals `numbersarray[i]` 
E.g. without the `shift()` method the example 
`TwoSumNumbers(10, [5,3,4,6,8])` would return `true` when it's expected to return `false` since no two values in the array can add up to give `10`.
This is because for 
`number - numbersarray[0]`,  the  `summation` equals `5` in this case, and without the `shift()` method `array.includes(summation)` returns `true`
Since 5 is still in the numbersarray.